mastodon/Mastodon.py

```diff
@@ -818,46 +819,63 @@ def fetch_remaining(self, first_page):
     ###
     # Streaming
     ###
-    def user_stream(self, listener):
+    def user_stream(self, listener, async=False):
```

Comment:
With the old behavior, a call to any of the streaming API functions blocks indefinitely, and the only way the consumer is ever called again on that thread is through the callbacks. I'm not sure I would call that asynchronous.

Perhaps a better term would be concurrent?